Tests/SentryTests/Integrations/SentrySessionTrackerTests.swift

```diff
@@ -28,15 +28,22 @@ class SentrySessionTrackerTests: XCTestCase {
     }
     
     private var fixture: Fixture!
+    private var sut: SessionTracker!
 
     override func setUp() {
         super.setUp()
 
         fixture = Fixture()
+        sut = fixture.getSut()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        sut.stop()
     }
 
     func testStartClosesPreviousCachedSession() {
-        fixture.getSut().start()
+        sut.start()
```

Comment:
M: if the idea is to not DRY, why not starting on `setUp` as well?

Reply:
Because `testStartClosesPreviousCachedSession` needs to set some state before `start`.